cmd/exchange_template/rest.tmpl

```diff
@@ -0,0 +1,51 @@
+{{define "rest"}}
+package {{.Name}}
+
+import (
+	"context"
+	"net/http"
+
+	exchange "github.com/thrasher-corp/gocryptotrader/exchanges"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/request"
+)
+
+// Exchange implements exchange.IBotExchange and contains additional specific api methods for interacting with {{.CapitalName}}
```

Comment:
Suggestion/optional: break up rest -> rest_public, rest_private/auth, along with websocket, subscription with public and private files with there own test files and wrapper with its own test file. Can also add in rate_limits.go with its own test file as well. This can be done in another PR if preferred.

Reply:
Thank you again for the suggestion here. I think we should keep this option for when we are forced to split these endpoints through the template.

As you know not all exchanges use a separate HTTP requester and I believe we can leave a comment suggesting that kind of implementation in the rest as I have done; You can check it in the main file.